alpha_importer.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function
+from redis import Redis
+import config
+import requests
+import ujson
+import time
+import sys
+
+redis_connection = Redis(host = config.REDIS_HOST)
+QUEUE_KEY = 'alphaimporter:queue'
+SET_KEY_PREFIX = 'alphaimporter:set'
+ALPHA_URL = 'https://api.listenbrainz.org/'
+BETA_URL = 'http://0.0.0.0:3031'
```

Comment:
Let's just pick what we plan to do:

ALPHA_URL: https://alpha.listenbrainz.org
BETA_URL: https://listenbrainz.org